src/Vezeeta.Application/Features/Patients/PatientMapper.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using AutoMapper;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+
+using Vezeeta.Application.Features.Patients.DTOs;
+using Vezeeta.Domain.Entities;
+
+namespace Vezeeta.Application.Features.Patients;
+public class PatientMapper : Profile
+{
+    public PatientMapper() 
+    {
+        CreateMap<PatientRegisterDto, ApplicationUser>()
+               .ForMember(dest => dest.FullName, opt => opt.MapFrom(src => $"{src.FirstName} {src.LastName}"))
+               .ForMember(dest => dest.Email, opt => opt.MapFrom(src => src.Email))
+               .ForMember(dest => dest.PasswordHash, opt => opt.MapFrom(src => HashPassord(src.Password)))
```

Comment:
### Anti-Pattern Alert
*Never* leak your business logic into the mappers. Mappers have only one task: Take object A values and place them in object B values. If you have any logic, it should be implemented in your Handler.